src/test/java/com/mongodb/jdbc/BsonTypeInfoTest.java

```diff
@@ -20,7 +20,11 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import java.nio.charset.StandardCharsets;
 import java.sql.SQLException;
+import org.bson.*;
```

Comment:
In the past, I know we've tried to avoid wildcard imports. I'm not sure if this is something asserted by spotless, or if it something we want to continue to do. I believe no other files have wildcard imports but we can check that. If we do have them in other places, then there is nothing to do here. If we do not use them or decide we do not want them, can you replace this with the actual imports needed?

You can also configure your IDE/editor to not use wildcard imports when there are multiple classes imported from the same package!